BrainPortal/app/controllers/users_controller.rb

```diff
@@ -356,6 +356,16 @@ def send_password #:nodoc:
     @user = User.where( :login  => params[:login], :email  => params[:email] ).first
 
     if @user
+      if user_must_link_to_globus?(@user)
+        contact = RemoteResource.current_resource.support_email.presence || User.admin.email.presence || "the support staff"
+        wipe_user_password_after_globus_link(@user)  # for legacy or erroneously set users
```

Comment:
This method take 2 arguments now, the oidc (not available in this case) and the user. 

We can do something like that: 

```
wipe_user_password_after_oidc_link("send_password", @user)
```

and 

```
   def wipe_user_password_after_oidc_link(oidc, user)
     wipe_by = oidc.is_a?(String) ? "Wiped-By-#{oidc}-Link-" : "Wiped-By-#{oidc.name}-Link-"
     user.update_attribute(:crypted_password, wipe_by + User.random_string)
     user.update_attribute(:salt            , wipe_by + User.random_string)
     user.update_attribute(:password_reset  , false)
   end
```